core_lib/src/interface/timelinecells.cpp

```diff
@@ -195,8 +192,8 @@ void TimeLineCells::drawContent()
 
     Q_ASSERT(object != nullptr);
 
-    const Layer* layer = mEditor->layers()->currentLayer();
-    if (layer == nullptr) return;
+    const Layer* currentLayer = mEditor->layers()->currentLayer();
```

Comment:
Btw, the reason for this change is the declaration of another identically named variable on line 344. However renaming this variable instead seemed a bit more natural to me.